EasyTipView.xcodeproj/project.pbxproj

```diff
@@ -401,6 +418,7 @@
 		1310ECA41D0C53800000E71E /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
I wish this pull request to be merged, however I think this setting should not be added.